package rec.app;

import "rec/command/Command.proto";

message MenuEntry {
  optional uint32 command = 1;
  optional string submenu = 2;
};

message Menu {
  optional command.Description description = 1;
  repeated MenuEntry entry = 2;
};

message Menus {
  repeated Menu menu = 1;
};

message MenuBar {
  optional command.Description description = 1;
  repeated string menu = 2;
};

message MenuCollection {
  optional command.Description description = 1;
  repeated MenuBar menu_bar = 2;
};
